Created two sections navbar & headline

Mofuhidy opened the pull request:
created the 2 sections that contain a navbar and headline that has a small paragraph about me

Mofuhidy committed:
added the project files

Mofuhidy committed:
update readme file with all information

Mofuhidy committed:
Create MIT license file

Mofuhidy committed:
update project name and description

Zuhaib042 reviewed (commented):
Hi @Mofuhidy 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has ambiguous sub-sections under `Getting Started` section in the readme. kindly consider filling them with appropriate and relevant information

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Mofuhidy committed:
add information to getting started section

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,49 @@
+</head>
+
+<body>
+    <nav class="navBar">
+        <img src="imgs/My Logo.svg" alt="logo" class="logo">
+        <img src="imgs/Union.svg" alt="menu burger" class="menuIcon">
+        <ul class="navInfo">
+            <li>Portoflio</li>
+            <li>About</li>
+            <li>Contact</li>
+            <li><img src="imgs/message.svg" alt="message incon"></li>
+        </ul>
+    </nav>
```
- [x] I would suggest you please wrap this section with the `header` tag typically the header tag is used as a page header.

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,49 @@
+</head>
+
+<body>
+    <nav class="navBar">
+        <img src="imgs/My Logo.svg" alt="logo" class="logo">
```
- [x] I would suggest you please wrap the logo with an `a` tag for the convention, semantics, and user experience(UX) reason.

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,49 @@
+<body>
+    <nav class="navBar">
+        <img src="imgs/My Logo.svg" alt="logo" class="logo">
+        <img src="imgs/Union.svg" alt="menu burger" class="menuIcon">
+        <ul class="navInfo">
+            <li>Portoflio</li>
+            <li>About</li>
+            <li>Contact</li>
+            <li><img src="imgs/message.svg" alt="message incon"></li>
+        </ul>
```
- [x] I would suggest you please wrap only the hamburger button with the `nav` tag that is on the right for semantics reasons.
- [ ] [Optional]  Also remove the `ul` and `li` tags for now you can use this on the desktop version.

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,49 @@
+            <li><img src="imgs/message.svg" alt="message incon"></li>
+        </ul>
+    </nav>
+    <main>
+        <div class="mainParagraph">
+            <h1 class="welcomeText">
+                Hey There.
+                I’m Fuhidy.
+            </h1>
+            <h3 class="supportingText">
+                I'm a Software Developer
+            </h3>
+            <p class="description">
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you
+                see and have a project you need coded,
+                don’t hesitate to contact me.
+            </p>
+        </div>
+        <div class="social-media">
+            <a href="#"> <img src="imgs/social-media/github.svg" alt="github logo"></a>
+            <a href="#"> <img src="imgs/social-media/linkedin.svg" alt="linkedin logo"></a>
+            <a href="#"> <img src="imgs/social-media/angiel.svg" alt="angelist logo"></a>
+            <a href="#"> <img src="imgs/social-media/twitter.svg" alt="twitter logo"></a>
+            <a href="#"> <img src="imgs/social-media/medium.svg" alt="medium logo"></a>
+        </div>
+    </main>
```
- [x] I would suggest you please use the section tag because the section represents a generic standalone section of the document. please use the section tag on it.

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,49 @@
+                don’t hesitate to contact me.
+            </p>
+        </div>
+        <div class="social-media">
+            <a href="#"> <img src="imgs/social-media/github.svg" alt="github logo"></a>
+            <a href="#"> <img src="imgs/social-media/linkedin.svg" alt="linkedin logo"></a>
+            <a href="#"> <img src="imgs/social-media/angiel.svg" alt="angelist logo"></a>
+            <a href="#"> <img src="imgs/social-media/twitter.svg" alt="twitter logo"></a>
+            <a href="#"> <img src="imgs/social-media/medium.svg" alt="medium logo"></a>
+        </div>
```
- [x] This is the list of the icons and for that, we use the `ul` tag to group them `li` per icon and each icon has a link (`a` tag) has is done. please implement it

roshan-bajgain reviewed (changes requested):
Hi @Mofuhidy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Nice design ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mofuhidy committed:
change nav tag to header tag

Mofuhidy committed:
wrap the img logo with anchor tag

Mofuhidy committed:
wrap the hamburger menu with <nav>

Mofuhidy committed:
fix the styling of social media icons

change main tag to section

Mofuhidy committed:
change main tag to section

fix the group tag of social media icons

Mofuhidy committed:
Merge branch 'created-two-sections' of https://github.com/Mofuhidy/my-portfolio into created-two-sections
merge the edits that given to me by the reviewer

Mofuhidy committed:
delete empty line before rule

AbelG101 reviewed (approved):
![](https://media.giphy.com/media/YPI7uEvEZr7siYRlZD/giphy.gif)


# Hi @Mofuhidy 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- UI is looking fabulous, on point ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title ✔

## Optional suggestions
- [x] I'd suggest you improve your PR description to be more detailed and specific on what features you've added you can follow the below template for your next PR
```
### Changes
- Added toolbar with logo and menu icon
- Added a landing page for my portfolio
    - Added a header with my name and role.
    .
    .
    .
```
- The "###" before the "Changes" text is to give it a heading (level 3)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mofuhidy commented:
Okay thank you very much for your advice😊Sent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: Abel Gebeyehu ***@***.***> Date: 4/26/23  8:27 PM  (GMT+03:00) To: Mofuhidy/my-portfolio ***@***.***> Cc: Mofuhidy ***@***.***>, Mention ***@***.***> Subject: Re: [Mofuhidy/my-portfolio] Created two sections navbar & headline (PR #1) 
@AbelG101 approved this pull request.


Hi @Mofuhidy 👋,
Your project is complete!  There is nothing else to say other than... it's time to merge it.
Congratulations! 🎉 🥳🥇
Highlights

UI is looking fabulous, on point ✔
Well-structured files ✔
All linter checks are passing ✔
Descriptive PR title ✔

Optional suggestions

 I'd suggest you improve your PR description to be more detailed and specific on what features you've added you can follow the below template for your next PR

### Changes
- Added toolbar with logo and menu icon
- Added a landing page for my portfolio
    - Added a header with my name and role.
    .
    .
    .


The "###" before the "Changes" text is to give it a heading (level 3)

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
